lib/serialize.ml

```diff
@@ -537,6 +537,21 @@ module Writer = struct
         ~has_priority:false
         faraday)
 
+  let write_response_trailers t hpack_encoder frame_info trailers =
+    if not (is_closed t.encoder) then (
+      let faraday = Faraday.of_bigstring t.headers_block_buffer in
+      (* From RFC7540§8.1:
+       *  optionally, one HEADERS frame, followed by zero or more
+       *  CONTINUATION frames containing the trailer-part, if present (see
+       *  [RFC7230], Section 4.1.2). *)
+      encode_headers hpack_encoder faraday trailers;
```

Comment:
Note to self: we could probably unify this code with the request / response header serializers above.